ui/.env.development

```diff
@@ -3,3 +3,4 @@ REACT_APP_VERSION=$npm_package_version
 REACT_APP_NAME=$npm_package_name
 REACT_APP_SENTRY_DSN=https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sentry.io/5721090
 REACT_APP_SENTRY_ENABLED=false
+RELEASE_VERSION=$set_by_config_overrides
```

Comment:
Maybe no need to set this one?

Reply:
I would like to have a single source for environment variables. So this is something like a comment.
Maybe `RELEASE_VERSION=DO_NOT_EDIT_it_set_in_config-overrides.js` will be better? Or just delete this one.

Reply:
How about `REACT_APP_RELEASE_VERSION=unknown` so that when something gets wrong we have at least a "unknown" value?

Reply:
It would be good if we can write some comments here. But it depends on how dotenv works.